src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
```

Comment:
The selector '.stars--1 :nth-child(1)' has a space before ':nth-child', which makes it a descendant selector. This means it will select any element that is the first child of any element inside '.stars--1'. If you intended to select the first child of '.stars--1' directly, remove the space to make it '.stars--1:nth-child(1)'.